Cargo.lock

```diff
@@ -12,9 +12,9 @@ dependencies = [
 
 [[package]]
 name = "addr2line"
-version = "0.12.1"
+version = "0.12.0"
```

Comment:
I'm not sure why Cargo.lock contains quite a few downgrades given that we didn't change Cargo.toml. Can you check that this is not a leftover change after a modification that might have been rolled back?

Reply:
Not sure why git is showing these changes. The current version in master is 0.12.0 not 0.12.1.

Reply:
I'll check that this file is not modified in `master` after the merge - a bit baffled as to why it is showing up here.